### ISSUE TITLE
not working anymore

### ISSUE THREAD

**Issue by ValerioTomaino** [kylon/Sharedown#49]
today i tried downloading some videos from sharepoint in the same way and settings of the past and it's not working. it opens the video page and can't load it. if i open the video myself, i can see it

**Comment by kylon** [comment]
any error?
keep in mind the video will not play on sharedown, thats how it works

it seems they are moving everyone to stream on sharepoint now, but download should work and it works here too

**Comment by ValerioTomaino** [comment]
nope, no errors from sharedown, seems like it just can't download it unless i do the previous trick

**Comment by kylon** [comment]
is the link the correct one for sharedown?

**Comment by ValerioTomaino** [comment]
yup link one, took it in the same way as always: https://uniroma3.sharepoint.com/:v:/r/sites/AA2122-FONDAMENTIDIAUTOMATICA-20801778PANZIERI/Documenti%20condivisi/General/Recordings/Solo%20visualizzazione/General-20220406_140706-Registrazione%20della%20riunione.mp4?csf=1&web=1&e=HXRBgS

this is the second link (working one) took it opening the video in sharepoint from another browser. if i paste this after the log in in sharedown, it works
https://uniroma3.sharepoint.com/sites/AA2122-FONDAMENTIDIAUTOMATICA-20801778PANZIERI/Documenti%20condivisi/Forms/AllItems.aspx?csf=1&web=1&e=EzgUV7%2F&FolderCTID=0x0120005A4E1F6F1F5A7F4D919F47C68AE4795A&sortField=Modified&isAscending=true&id=%2Fsites%2FAA2122%2DFONDAMENTIDIAUTOMATICA%2D20801778PANZIERI%2FDocumenti%20condivisi%2FGeneral%2FRecordings%2FSolo%20visualizzazione%2FGeneral%2D20220406%5F140706%2DRegistrazione%20della%20riunione%2Emp4&parent=%2Fsites%2FAA2122%2DFONDAMENTIDIAUTOMATICA%2D20801778PANZIERI%2FDocumenti%20condivisi%2FGeneral%2FRecordings%2FSolo%20visualizzazione

**Comment by kylon** [comment]
i cant repro, it seems a redirect happening after your login

what login method?
have you tried to enable chrome data folder and retry download after the first fail?

should log url changes to log file to see if we can find when it goes wrong
[Sharedownt.zip](https://github.com/kylon/Sharedown/files/9504639/Sharedownt.zip)

**Comment by ValerioTomaino** [comment]
the login is microsoft one for institutional account, for redirections i don't know, everything is enabled i'll try get the logs for you

**Comment by sato-shi** [comment]
I saw similar issues. I changed "preload.js" to use Google Chrome instead of Chromium as browser in Puppeteer. Now I can download again.

`
function _getPuppeteerArgs(puppyExePath, userdataFold) {
        const pargs = {
            //executablePath: _getPuppeteerExecutablePath(puppyExePath),
            executablePath:"/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome",
            headless: false,
            args: ['--disable-dev-shm-usage']
        };
```

**Comment by tkgmomosheep** [comment]
> I saw similar issues. I changed "preload.js" to use Google Chrome instead of Chromium as browser in Puppeteer. Now I can download again.
> 
> ```
> function _getPuppeteerArgs(puppyExePath, userdataFold) {
>         const pargs = {
>             //executablePath: _getPuppeteerExecutablePath(puppyExePath),
>             executablePath:"/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome",
>             headless: false,
>             args: ['--disable-dev-shm-usage']
>         };
> ```
For Windows, use the code snippet below
```     
    function _getPuppeteerArgs(puppyExePath, userdataFold) {
        const pargs = {
          executablePath:
            "C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe",
          headless: false,
          args: ["--disable-dev-shm-usage"],
        };
```

**Comment by kylon** [comment]
weird, i still see no error on my pc
maybe i ll add an option to set chrome as browser

**Comment by BorisDeGraaff** [comment]
> today i tried downloading some videos from sharepoint in the same way and settings of the past and it's not working. it opens the video page and can't load it. if i open the video myself, i can see it.
> 
> edit: this is the error page ![Immagine 2022-09-07 101908](https://user-images.githubusercontent.com/94002109/188828077-12d3968d-3a1f-43ff-9812-40daa19c0902.png) if from here i copy the link of the video opened in another browser (same link i think), it works

I have the same issue, how does your trick exactly work? Opening the link in another chrome browser does nothing for me.

**Comment by ValerioTomaino** [comment]
Open the video page on another browser then, when you're stuck with Chrome puppet, copy paste the link inside the Chrome puppet. For me worked

Inviato da Yahoo Mail su Android 
 
  Il lun, 31 ott, 2022 alle 12:23, ***@***.***> ha scritto:   



today i tried downloading some videos from sharepoint in the same way and settings of the past and it's not working. it opens the video page and can't load it. if i open the video myself, i can see it.

edit: this is the error page  if from here i copy the link of the video opened in another browser (same link i think), it works


I have the same issue, how does your trick exactly work? Opening the link in another chrome browser does nothing for me.

**Comment by brianchen0** [comment]
Tried to download using Chrome instead of Chromium, still shows the error as below.

![image](https://user-images.githubusercontent.com/103989240/200236921-adec6ff1-1ba2-4c7e-afc0-abde97550b72.png)

![image](https://user-images.githubusercontent.com/103989240/200236943-4fb813a8-2bfe-4923-ace9-b2de674c107c.png)

**Comment by kylon** [comment]
> Tried to download using Chrome instead of Chromium, still shows the error as below.
> 
> ![image](https://user-images.githubusercontent.com/103989240/200236921-adec6ff1-1ba2-4c7e-afc0-abde97550b72.png)
> 
> ![image](https://user-images.githubusercontent.com/103989240/200236943-4fb813a8-2bfe-4923-ace9-b2de674c107c.png)

this is a different thing, see #54

**Comment by brianchen0** [comment]
> > Tried to download using Chrome instead of Chromium, still shows the error as below.
> > ![image](https://user-images.githubusercontent.com/103989240/200236921-adec6ff1-1ba2-4c7e-afc0-abde97550b72.png)
> > ![image](https://user-images.githubusercontent.com/103989240/200236943-4fb813a8-2bfe-4923-ace9-b2de674c107c.png)
> 
> this is a different thing, see #54

Got it, thanks Kylon

**Comment by kylon** [comment]
please try 5.2.1 release

**Comment by kylon** [comment]
closing for inactivity, should be fixed in latest release
